pkg/lua/busted/busted.go

```diff
@@ -0,0 +1,15 @@
+package busted
+
+import (
+	"github.com/pinpt/dialect"
+	"regexp"
+)
+
+var (
+	containsDescribe = regexp.MustCompile("describe\\s*\\(")
+	containsIt       = regexp.MustCompile("it\\s?(\\.only|skip)?\\s?\\(")
+)
+
+func IsTest(line *dialect.DialectLine) bool {
+	return containsDescribe.MatchString(line.Buffer) && containsIt.MatchString(line.Buffer)
+}
```

Comment:
busted uses the same syntax as mocha, so I used the same code from mocha. or am I missing your question?